rest-client/src/main/java/io/apicurio/registry/client/RegistryRestClient.java

```diff
@@ -34,7 +34,7 @@
     ArtifactMetaData createArtifact(ArtifactType xRegistryArtifactType, String xRegistryArtifactId, IfExistsType ifExists, InputStream data);
 
 
-    Response getLatestArtifact(String artifactId);
+    ResponseBody getLatestArtifact(String artifactId);
```

Comment:
I was thinking maybe `byte[]` or `InputStream` or `ContentHandle` here.  Is there something that the okhttp3 class gives us that's better than just plain data?

Reply:
You can get the `InputStream` from the `ResponseBody` class and also the content length and type.